app/docker/views/containers/logs/containerLogsController.js

```diff
@@ -33,7 +38,7 @@ function ($scope, $transition$, $interval, ContainerService, Notifications) {
   function setUpdateRepeater() {
     var refreshRate = $scope.state.refreshRate;
     $scope.repeater = $interval(function() {
-      ContainerService.logs($transition$.params().id, 1, 1, 0, $scope.state.lineCount)
+      ContainerService.logs($transition$.params().id, 1, 1, $scope.state.displayTimestamp, $scope.state.lineCount)
```

Comment:
Please use a ternary form here: `$scope.state.displayTimestamps ? 1 : 0`

Even if passing a true parameter to the API works, it actually expects either 0 or 1 values.